canu/generate/switch/config/config.py

```diff
@@ -71,18 +71,42 @@
 
 # ttp preserve templates
 # pulls the interface and lag from switch configs.
-aruba_template = """
-interface {{ interface }}
-    lag {{ lag }}
-"""
-dell_template = """
-interface ethernet{{ interface }}
-  channel-group {{ lag }}
-  channel-group {{ lag }} mode active
-"""
-mellanox_template = """
-interface ethernet {{ interface }} {{ _line_ | contains("channel-group") }} {{ lag }} mode active
-"""
+aruba_template = path.join(
```

Comment:
For future ideation.... This is quite a bit of cut-n-paste code.  you could create a dictionary of paths and loop over it..kinda like:

```
templates = defaultdict()
for vendor in ['aruba', 'dell', 'mellanox']':
    templates[vendor] = path.join("canu", "generate"........,f"{vendor}_lag.txt")